.github/drafts/16521/Code-scanner-Draft.md

```diff
@@ -0,0 +1,122 @@
+|-----------------|----------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| Text            | Formatting           | Sets the label text of the widget.                                                                                  | -                           |
+| [Visible](https://docs.appsmith.com/reference/widgets)          | Binding & Formatting | Controls widget's visibility on the page. When turned off, the widget won't be visible when the app is published. Learn more about [Visible Property](https://docs.appsmith.com/reference/widgets)   | {{CodeScanner1.isVisible}}  |
+| [Disabled](https://docs.appsmith.com/reference/widgets)         | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input won't be allowed. Learn more about [Disable Property](https://docs.appsmith.com/reference/widgets)      | {{CodeScanner1.isDisabled}} |
+| [Animate Loading](https://docs.appsmith.com/reference/widgets)  | Formatting           | Controls the loading of the widget. Learn more about [Tooltip](https://docs.appsmith.com/reference/widgets)                                                                                 | -                           |
+| [Animate Loading](https://docs.appsmith.com/reference/widgets)           | Formatting           | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user. Learn more about [Tooltip](https://docs.appsmith.com/reference/widgets)    | -                           |
+| value           | Binding              | Fetches the value of scanned code                                                                                   | {{CodeScanner1.value}}      |
+|                 |                      |                                                                                                                     |                             |
+
+
+
+### Text
+
+Text/Label is a widget property that lets you set the text inside the button, describing the function it performs. It's used to display static text on the button. For example, you can enter "Scan Menu" as the label for a button that scans the restaurant menu.
+
+[Text Video](https://youtu.be/o80-IKcXAVQ)
+
+
+### Value
+
+Value is a binding property, it fetches the response from the scanned code. To bind the value of the code scanner widget to another widget, open the property pane of it, and add the code snippet given below:
+
+```
+{{<your_widget>.value}}
+```
+
+
+For example, let's take a Code Scanner widget(CodeScanner1) and bind its value to a text widget. Drag a text widget onto the canvas and add the following code:
+
+```
+{{CodeScanner1.value}}
+```
+
+.https://youtu.be/zfkpIzaiTX0
+
+
+# Events
+
+They are a set of actions that you can perform on the widget. The following table lists the actions:
```

Comment:
This sentence looks a bit confusing. Events are not actions. AFAIK, in Appsmith, Events are triggered usually by **a user action** and an Action is what you do when an event is triggered, eg: run a query/JS, navigate to another page,... We'll need to frame this in a better way